jwst/rscd/rscd_step.py

```diff
@@ -32,6 +31,18 @@ def process(self, input):
                     result.meta.cal_step.rscd = 'SKIPPED'
                     return result
 
+
+                # Check that data has the minimum number of groups/int
+                sci_ngroups = input_model.data.shape[1]     # number of groups
+                min_number = 4
+                if sci_ngroups < min_number :
+                    self.log.warning('Input file does not contain enough groups '
+                                     ' for RSCD correctionto be applied ')
```

Comment:
don't need whitespace at the end of line 39 and at the beginning of line 40 (can remove one of them)